pom.xml

```diff
@@ -462,10 +468,22 @@
                 <artifactId>netty-handler</artifactId>
                 <version>${version.netty}</version>
             </dependency>
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-buffer</artifactId>
+                <version>${version.netty}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty</artifactId>
+                <version>3.10.6.Final</version>
+            </dependency>
 
             <!-- ReactiveX -->
             <dependency>
                 <groupId>io.reactivex</groupId>
+
+
```

Comment:
Should probably remove this extraneous whitespace.

Reply:
Fixed.